packages/asset-kit/src/settings.ts

```diff
@@ -0,0 +1,83 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { Color, SettingBlock, defineSettings } from '@frontify/guideline-blocks-settings';
+import {
+    Radius,
+    getBorderRadiusSettings,
+    getBorderRadiusSlider,
+    getBorderSettings,
+} from '@frontify/guideline-blocks-shared';
+
+export const ASSET_SETTINGS_ID = 'images';
+export const BACKGROUND_COLOR_DEFAULT_VALUE: Color = {
+    red: 241,
+    green: 241,
+    blue: 241,
+    alpha: 1,
+};
+export const BORDER_COLOR_DEFAULT_VALUE: Color = {
+    red: 230,
+    green: 230,
+    blue: 230,
+    alpha: 1,
+};
+
+const backgroundColor = (id: string, defaultValue = true): SettingBlock => ({
+    id: `hasBackground_${id}`,
+    label: 'Background',
+    type: 'switch',
+    defaultValue,
+    on: [
+        {
+            id: `backgroundColor_${id}`,
```

Comment:
Please use camelCase for all setting ids

Reply:
Changed all that where possible without touching the shared. 

All the settings helpers that are used are adding the id's in a non camelCase way, if we change it there we have breaking changes in other blocks so i assume the best is to leave it like that.


All settings helpers are creating ids in a non camelCase way if you add custom id (if you use it multiple time within the same setting you have to)

Reply:
Makes sense, thanks!